tests/unit/components/enhanced-table/filter-row/filter-row.spec.jsx

```diff
@@ -0,0 +1,65 @@
+import FilterRow from '~/components/enhanced-table/filter-row/FilterRow'
+import { useTableContext } from '~/context/table-context'
+import { vi } from 'vitest'
+import { fireEvent, screen, render, renderWithProviders } from '@testing-library/react'
```

Comment:
Set the correct order of imports

And name this file correctly 'FilterRow.spec.jsx'

Reply:
Fixed